py3status/modules/process_status.py

```diff
@@ -43,29 +63,30 @@ def _get_text(self):
 
         if subprocess.call(pgrep,
                            stdout=fnull, stderr=fnull) == 0:
-            text = self.format_running
             color = self.py3.COLOR_GOOD
+            text = self.py3.safe_format(self.icon_on,
+                                        {'process': self.process})
```

Comment:
this should just be `icon = self.icon_on`

`icon` is a better name than `text`